esphomeRemote/esphomeRemoteLight.h

```diff
@@ -2,30 +2,219 @@
+                                                          };
+        setAttribute(data);
+    }
+
+    void incBrightness() {
+        if(!is_brightness_in_sync){
+            return;
+        }
+        is_brightness_in_sync = false;
+        const std::map< std::string, std::string > data = {
+                                                              {"entity_id",entityId.c_str()},
+                                                              {"brightness", to_string(local_brightness + id(inc_brightness_step))} ,
+                                                          };
+        setAttribute(data);
+    }
+
+    void setAttribute(const std::map< std::string, std::string> &data){
+        call_homeassistant_service("light.turn_on", data);
+    }
 
     void toggleLight() {
       call_homeassistant_service("light.toggle", {
         {"entity_id", entityId.c_str()}
       });
     }
 
+    bool supportsBrightness(){
+        return color_mode != onoff_type && color_mode != unknown_type;
+    }
+
+    bool supportsColorTemperature(){
+        return color_mode == color_temp_type || color_mode == rgbww_type;
+    }
+
+    std::vector<std::shared_ptr<MenuTitleBase>> lightTitleItems() {
+      std::vector<std::shared_ptr<MenuTitleBase>> out;
+        out.push_back(std::make_shared<MenuTitleBase>(friendlyName, entityId, onState ? OnMenuTitleState : OffMenuTitleState));
+
+        std::stringstream ss;
+        int width_available = id(display_size_x) - 2 * id(slider_margin_size);
+        if(supportsBrightness()){
+            float slider_factor = 1;
+            if(local_brightness > 0){
+                float percent = ((float)local_brightness/255.0);
+                int percentInt = (int)(percent*100);
+                ss << "Brightness - " << percentInt << " %%";
+                slider_factor = width_available / MAX_BRIGHTNESS;
+            }else{
+                ss << "Brightness";
+            }
+            out.push_back(std::make_shared<MenuTitleSlider>("Brightness", ss.str(), entityId, onState ? OnMenuTitleState : OffMenuTitleState, (int)(local_brightness * slider_factor)));
+            ss.str("");
+        }
+        if(supportsColorTemperature()){
+            float slider_factor = 1;
+            if(local_color_temp > 0){
+                ss << "Temperature - " << 1000000/local_color_temp << " K ";
+                slider_factor = width_available / (max_mireds - min_mireds);
+            }else{
+                ss << "Temperature";
+            }
+            out.push_back(std::make_shared<MenuTitleSlider>("Temperature", ss.str(), entityId, onState ? OnMenuTitleState : OffMenuTitleState, (int)(local_color_temp-min_mireds * slider_factor)));
+            ss.str("");
+        }
+      return out;
+    }
+
   private:
     void state_changed(std::string newOnState) {
-      ESP_LOGI("Light", "state changed to %s", newOnState.c_str());
+      ESP_LOGI("brightness", " changed to %s", newOnState.c_str());

```

Comment:
this should be "State" instead of brightness. also can you add the entity id to all of these logs so it's clear which light